src/Sulu/Bundle/AdminBundle/Resources/js/components/Table/Row.js

```diff
@@ -1,13 +1,39 @@
 // @flow
 import React from 'react';
+import type {ChildrenArray} from 'react';
 import Checkbox from '../Checkbox';
 import {Radio} from '../Radio';
-import type {ButtonConfig, RowProps} from './types';
+import type {ButtonConfig, SelectMode} from './types';
 import Cell from './Cell';
 import ButtonCell from './ButtonCell';
 import tableStyles from './table.scss';
 
-export default class Row extends React.PureComponent<RowProps> {
+type Props = {
+    children: ChildrenArray<*>,
```

Comment:
Shouldn't we be able to type this more specific? It only allows certain types of `Cell`s, doesn't it?